.github/workflows/README.md

```diff
@@ -0,0 +1,71 @@
+| Test with Conda             | .github/workflows/ci_test-conda.yml | Same as ci_test-full.yml but with dependencies installed with conda.                                                                                                                        | CPU           | (3.8, 1.8), (3.8, 1.9), (3.8, 1.10), (3.9, 1.11) | linux               |
+| Test slow                   | .github/workflows/ci_test-slow.yml  | Run only slow tests. Slow tests usually need to spawn threads and cannot be speed up or simplified.                                                                                         | CPU           | (3.7, 1.8)                                       | linux, mac, windows |
+| PL.pytorch-lightning (IPUs) | .azure-pipelines/ipu-tests.yml      | Run only IPU-specific tests.                                                                                                                                                                | IPU           | (3.8, 1.9)                                       | linux               |
+| PL.pytorch-lightning (HPUs) | .azure-pipelines/hpu-tests.yml      | Run only HPU-specific tests.                                                                                                                                                                | HPU           | (3.8, 1.10)                                      | linux               |
+| PL.pytorch-lightning (GPUs) | .azure-pipelines/gpu-tests.yml      | Run GPU-specific tests as well as non-accelerator-specific, standalone and example tests. Each standalone test needs to run individually to avoid unwanted interactions between test cases. | GPU           | (3.7, 1.8)                                       | linux               |
+| PyTorchLightning.Benchmark  | .azure-pipelines/gpu-benchmark.yml  | Run speed/memory benchmarks against pure PyTorch.                                                                                                                                           | GPU           | (3.7, 1.8)                                       | linux               |
+| test-on-tpus                | .circleci/config.yml                | Run only TPU-specific tests.                                                                                                                                                                | TPU           | (3.7, 1.9)                                       | linux               |
+
+- \*Accelerators used in CI
+  - GPU: 2 x NVIDIA P100
+  - TPU: Google GKE TPUv2/3
+  - IPU: [Colossus MK1 IPU](https://www.graphcore.ai/products/ipu)
+  - HPU: [AWS EC2 dl1.24xlarge](https://aws.amazon.com/ec2/instance-types/dl1/) which has 8 Gaudi accelerators
```

Comment:
Not confident about the IPU and HPU setups 👀

Reply:
HPU setup sounds right :)